web/src/DetailedInfoPage.tsx

```diff
@@ -372,6 +391,133 @@ export const DetailedInfoPage: React.FunctionComponent<{}> = () => {
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sequence Information
+                </h4>
+                {renderSeqInfo(sequenceInfo)}
+                {prepReadMetadata(sequenceInfo.meta || {})}
+            </>
+        );
+    };
+
+    const renderSampleSection = () => {
+        if (!sample || !sampleInfo) {
+            return <></>;
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sample Information
+                </h4>
+                {Object.entries(sampleInfo)
+                    .filter(([key, value]) =>
+                        sampleFieldsToDisplay.includes(key)
+                    )
+                    .map(([key, value]) =>
+                        value ? (
+                            <div key={`${key}-${value}`}>
+                                <b>{key}:</b> {value.toString()}
+                            </div>
+                        ) : (
+                            <React.Fragment
+                                key={`${key}-${value}`}
+                            ></React.Fragment>
+                        )
+                    )}
+            </>
+        );
+    };
+
+    const renderPedigreeSection = () => {
+        if (!sample || !pedigree) {
+            return <></>;
+        }
+        return (
+            <>
+                {pedigree
+                    .filter((item) => item.individual_id === selectedExternalID)
+                    .map((item) => (
+                        <React.Fragment
+                            key={`${item.individual_id}-${item.maternal_id}-${item.paternal_id}`}
+                        >
+                            {item.paternal_id &&
+                                item.maternal_id &&
+                                drawTrio(
+                                    item.paternal_id,
+                                    pedigree?.find(
+                                        (i) =>
+                                            i.individual_id === item.paternal_id
+                                    )!.affected,
+                                    item.maternal_id,
+                                    pedigree?.find(
+                                        (i) =>
+                                            i.individual_id === item.maternal_id
+                                    )!.affected,
+                                    item.individual_id,
+                                    item.affected,
+                                    item.sex
+                                )}
+                        </React.Fragment>
+                    ))}
+            </>
+        );
+    };
+
+    const renderTitle = () => {
+        return (
```

Comment:
Why not just set this to a component?
```suggestion
    const title = (
```

Reply:
Should be, fixed